src/chainstate/burn/operations/leader_block_commit.rs

```diff
@@ -767,7 +768,8 @@ impl LeaderBlockCommitOp {
     ) -> Result<SortitionId, op_error> {
         let tx_tip = tx.context.chain_tip.clone();
         let intended_sortition = match epoch_id {
-            StacksEpochId::Epoch21 => {
+            // TODO(sbtc): Verify that the 2.1 checks are sufficient here
```

Comment:
This TODO should be closed in this PR.  We should be 100% certain that the checks are sufficient here before merging.

Reply:
Agreed. I don't intend to merge anything until all of these TODO:s are either resolved or turned into issues.